Library/Homebrew/cmd/update.sh

```diff
@@ -321,6 +341,11 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  if [[ -n "$HOMEBREW_VERBOSE" ]]
+  then
+    oh1 "Fetching updates for all Git repositories"
```

Comment:
The output for each `$DIR` below only happens in non-parallel mode (notice the additional condition there), as it would otherwise just contribute to the intermingling of output. Thus this is the only line always being printed during the fetch phase. And since I'm about to rip out the `--no-parallel` stuff for now, the repository-specific output will have to go and this line probably should stay.